tck/rest/pom.xml

```diff
@@ -62,47 +62,31 @@
     </dependencies>
 
     <build>
-        <plugins>
-          <plugin>
-            <artifactId>maven-surefire-plugin</artifactId>
-            <version>2.19.1</version>
-            <configuration>
-              <systemPropertyVariables>
-                <org.apache.maven.user-settings>${session.request.userSettingsFile.path}</org.apache.maven.user-settings>
-              </systemPropertyVariables>
-              <includes>
-                <include>**/*IT.java</include>
-                <include>**/*Test.java</include>
-              </includes>
-              <environmentVariables>
-                <MP_METRICS_TAGS>tier=integration</MP_METRICS_TAGS>
-              </environmentVariables>
-            </configuration>
-          </plugin>
-
-          <plugin>
-            <artifactId>maven-failsafe-plugin</artifactId>
-            <configuration>
-              <includes>
-                <include>**/*IT.java</include>
-                <include>**/*Test.java</include>
-              </includes>
-              <systemPropertyVariables>
-                <org.apache.maven.user-settings>${session.request.userSettingsFile.path}</org.apache.maven.user-settings>
-              </systemPropertyVariables>
-              <environmentVariables>
-                <MP_METRICS_TAGS>tier=integration</MP_METRICS_TAGS>
-              </environmentVariables>
-            </configuration>
-            <executions>
-              <execution>
-                <goals>
-                  <goal>integration-test</goal>
-                  <goal>verify</goal>
-                </goals>
-              </execution>
-            </executions>
-          </plugin>
-        </plugins>
-    </build>
-</project>
\ No newline at end of file
+      <plugins>
```

Comment:
Again, why delete?  Eventually we will need these instructions.  I would comment them out for now, but leave them in the pom.xml.  If somebody completely new has to come along tomorrow to finish this up, we might as well leave some bread crumbs on what we were thinking...